packages/fluentui/react-northstar-fela-renderer/src/felaFocusVisibleEnhancer.ts

```diff
@@ -27,7 +12,7 @@ const felaFocusVisibleEnhancer = (renderer: Renderer) => {
   renderer._emitChange = (change: RendererChange) => {
     if (change.type === RULE_TYPE && change.selector.indexOf(':focus-visible') !== -1) {
       // Fela uses objects by references, it's safe to override properties
-      change.pseudo = change.pseudo ? change.pseudo.replace(':focus-visible', ':focus') : undefined;
+      change.pseudo = change.pseudo ? change.pseudo.replace(':focus-visible', ':focus') : '';
```

Comment:
🤔

Reply:
Strict typings are better as it should be a string and will default to an empty string:

https://github.com/robinweser/fela/blob/97148419e7941dbd5867b35df6ac4de6c5ed3e55/packages/fela/src/createRenderer.js#L224
https://github.com/robinweser/fela/blob/97148419e7941dbd5867b35df6ac4de6c5ed3e55/packages/fela/src/createRenderer.js#L302-L309